### PR TITLE
Icon and documentation for Enso VSCode extension

### PR DESCRIPTION
**Pull request by JaroslavTulach**
### Pull Request Description

Adds icon and rewords the main readme file for the Enso VSCode extension.

### Important Notes

Enso Icon in VSCode

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The [documentation has been updated](https://github.com/enso-org/enso/blob/f1b791a239e6a50467d47927acce6ba9653c08d6/tools/enso4igv/README.md)
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the style guides.
- All code has been tested:
  - [x] Manually tested

### CODE REVIEW

**Comment by JaroslavTulach** [comment]
CCing @ola-lis

File: engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionCallbacks.java [hunk 1]
```diff
@@ -145,7 +145,7 @@ private String typeOf(Object value) {
       resultType = Constants.UNRESOLVED_SYMBOL;
     } else {
       var typeOfNode = TypeOfNode.getUncached();
-      Object typeResult = typeOfNode.execute(value);
+      Object typeResult = value == null ? null : typeOfNode.execute(value);
```
Comment (4e6) [inline engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionCallbacks.java]: Seems unrelated :thinking:

**Review by 4e6 — APPROVED** [review]

File: engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionCallbacks.java (hunk 1 above)
Comment (JaroslavTulach) [reply engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionCallbacks.java]: Yup it is, sorry. However it is already already integrated as #8044, so there is no need to remove it.
